server/config/database.js

```diff
@@ -28,5 +28,6 @@ const queryDB = async (queryText, params) => {
 
 module.exports = {
     connectDB,
-    queryDB
+    queryDB,
+    pool,
```

Comment:
Exporting pool so we can make sure our write to the user and write to the profile both succeed.